packages/mui-material/src/Accordion/Accordion.js

```diff
@@ -73,6 +73,7 @@ const AccordionRoot = styled(Paper, {
         },
       },
       [`&.${accordionClasses.expanded}`]: {
+        overflowAnchor: 'auto', // keep active accordion in the viewport
```

Comment:
Does this have have any impact? I see no change.

Same question for `overflowAnchor: 'none', // Keep the same scrolling position` it seems that we don't need this anymore.

Reply:
> Does this have any impact? I see no change.

@oliviertassinari it does 

A) (before) all accordions have `overflowAnchor: 'none'`

https://github.com/user-attachments/assets/836fe5d1-13b4-48d4-8634-21b8ad1c205a

B) (after) all accordions have `overflowAnchor: 'none'` except **expanded** one

https://github.com/user-attachments/assets/e15764e2-bffd-4e85-9ad7-811be8b102df

C) no `overflowAnchor` at all

https://github.com/user-attachments/assets/7ccbf2b0-7d73-49b7-ba4e-70a72c0fb8c4

---

> Same question for `overflowAnchor: 'none', // Keep the same scrolling position` it seems that we don't need this anymore.

There is a slight difference in `B` and `C`  (after clicking second accordion)
In `B` you can see the first accordion
In `C` you can not see the first accordion


This was my proposal, I am okay with removing `overflowAnchor: 'none'`

Reply:
I saw joy ui's accordion and it does not have `overflowAnchor`
ig we can remove it

Reply:
B) looks better, indeed.

Reply:
would you like me to make any changes to the PR?